events/examplescenario/event.go

```diff
@@ -0,0 +1,138 @@
+package examplescenario
+
+import (
+	"fmt"
+	"github.com/sharovik/devbot/internal/helper"
+	"github.com/sharovik/devbot/internal/service/base"
+	"time"
+
+	"github.com/sharovik/devbot/internal/log"
+
+	"github.com/sharovik/devbot/internal/container"
+	"github.com/sharovik/devbot/internal/dto"
+)
+
+const (
+	//EventName the name of the event
+	EventName = "examplescenario"
+
+	//EventVersion the version of the event
+	EventVersion = "1.0.0"
+
+	helpMessage = "Ask me `write a message` and you will see the answer."
+
+	//The migrations folder, which can be used for event installation or for event update
+	migrationDirectoryPath = "./events/examplescenario/migrations"
+
+	regexChannel = `(?im)(?:<#)(\w+)(?:\|)`
+)
+
+//ExmplEvent the struct for the event object. It will be used for initialisation of the event in defined-events.go file.
+type ExmplSnrEvent struct {
```

Comment:
Is it a common practice in go to shorten names?

Reply:
Well
It's not a common practice, I will rename it to `EventStruct` name. Which will be shorter and more correct I think 🌝